### PR TITLE
NN-3527 Use prisonstaffhubclient token when making requests to the prisoner search service to get prisoner details

### PR DESCRIPTION
**Pull request by yasinmustafa**
In addition to this code change the GLOBAL_SEARCH role will need to be added to prisonstaffhubclient in auth

### CODE REVIEW

File: backend/tests/prisonerQuickLook.test.js [hunk 1]
```diff
@@ -67,7 +68,16 @@ describe('prisoner profile quick look', () => {
 
     offenderSearchApi.getPrisonersDetails = jest.fn().mockResolvedValue([])
 
-    controller = prisonerQuickLook({ prisonerProfileService, prisonApi, telemetry, offenderSearchApi, logError })
+    systemOauthClient.getClientCredentialsTokens = jest.fn().mockReturnValue({})
```
Comment (JamieDraperUK) [inline backend/tests/prisonerQuickLook.test.js:71]: Need to test that the systemOauthClient gets called

Comment (JamieDraperUK) [inline backend/tests/prisonerQuickLook.test.js:71]: Need to test that the systemOauthClient gets called

**Review by JamieDraperUK — DISMISSED** [review]
LGTM - missing unit tests

File: backend/tests/prisonerQuickLook.test.js (hunk 1 above)
Comment (yasinmustafa) [reply backend/tests/prisonerQuickLook.test.js:71]: Thanks Jamie, agreed added tests for this now

**Review by JamieDraperUK — APPROVED** [review]

**Pull request merged**